.vortex/tooling/src/helpers.php

```
@@ -776,12 +859,288 @@ function acli_exec(string $bin, string $subcommand, array $ctx, ?int &$exit_code
+
+  $data = json_decode((string) $response['body'], TRUE);
+  $items = (is_array($data) && isset($data['_embedded']) && is_array($data['_embedded']) && isset($data['_embedded']['items']) && is_array($data['_embedded']['items'])) ? $data['_embedded']['items'] : [];
+  $last = end($items);
+  $uuid = is_array($last) && is_string($last['uuid'] ?? NULL) ? $last['uuid'] : '';
+
+  if ($uuid === '') {
+    FAIL('Unable to retrieve an application UUID.');
+  }
+
+  return $uuid;
+}
```

PHPMD (2.15.0)

[warning] The function acquia_api_get_app_uuid() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)